### PR TITLE
flowblade: update to 2.6 & take ownership.

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/flowblade/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'flowblade'
 pkgname=flowblade
-version=2.4
-revision=2
+version=2.6
+revision=1
 archs=noarch
```
Comment (fosslinux) [inline srcpkgs/flowblade/template:5]: please remove noarch as part of the transition away from noarch

**Review by fosslinux — CHANGES_REQUESTED** [review]

File: srcpkgs/flowblade/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/flowblade/template:5]: Of course. Will be changed ASAP. Where can I read more about the transition away from noarch?

Comment (fosslinux) [reply srcpkgs/flowblade/template]: #24684, #24357

**Comment by shizonic** [comment]
Done.

**Comment by ericonr** [comment]
I had to launch it as `/usr/bin/flowblade` for it to work, because apprently having `/bin` in `PATH` broke its detection. Might want to report upstream.

It also doesn't support Wayland :/

**Comment by sgn** [comment]
> I had to launch it as `/usr/bin/flowblade` for it to work, because apparently having `/bin` in `PATH` broke its detection.

Fixed.

**Pull request merged**